### ISSUE TITLE
[Site] Download stats

### ISSUE THREAD

**Issue by mariotaku** [webosbrew/apps-repo#35]

**Comment by Informatic** [comment]
How are we going to make it work considering the whole site is static? Do we add some JS snippet to fill in that data? Or just update the counts on periodic repo updates?

Per-file download counts are exposed publicly on: https://api.github.com/repos/webosbrew/webos-homebrew-channel/releases (just noting it down)

**Comment by mariotaku** [comment]
@Informatic Yes by period repo updates. I'll create a task running everyday to collect download count in a window (e.g. 1 month)
